src/pages/Cart.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { remove } from '../Redux/slice/CardSlice'
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+
+
+const Cart = () => {
+  const product = useSelector((state) => state.card)
+  const dispatch = useDispatch()
```

Comment:
you can de structure dispatch from store directly and use that dispatch function